R/tribble.R

```diff
@@ -19,6 +30,26 @@
 #'   "c",   3
 #' )
 #'
+#' # Conversion expressions can be supplied, including additional
+#' # paremeters if needed.
+#' tribble(
+#'   ~colA, colB~factor(., levels = c("B", "A")),
```

Comment:
I'm fine with requiring the dot in `colA~as.character(as.factor(.))`, this is what magrittr does too. We could also be explicit about magrittr usage by requiring `~colA %>% as.character(as.factor(.))`, this would be better compatible with existing code.

So, we have the following options, all of which seem to be feasible to implement:

- `colA ~ as.character(as.factor(.))`
- `colA %>% as.character(as.factor(.))`
- `colA = as.character(as.factor(.))`

@hadley: Do you have a preference for specifying a conversion function for a column in `tribble()`?